README.md

```diff
@@ -8,45 +8,84 @@ Creates a scheduled Fargate Task in AWS
-| vpc_id                        | string                          | VPC ID to deploy the ECS fargate service and ALB                                                                                                                                                                                                      |                              |
-| private_subnet_ids            | list(string)                    | List of subnet IDs for the fargate service                                                                                                                                                                                                            |                              |
-| role_permissions_boundary_arn | string                          | ARN of the IAM Role permissions boundary to place on each IAM role created                                                                                                                                                                            |                              |
-| tags                          | map(string)                     | A map of AWS Tags to attach to each resource created                                                                                                                                                                                                  | {}                           |
+| Name                          | Type                            | Description                                                                                                                                                                    | Default                       |
+|-------------------------------|---------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------|
+| app_name                      | string                          | Application name to name your scheduled Fargate task and other resources                                                                                                       |                               |
+| existing_ecs_cluster          | [object](#existing_ecs_cluster) | Existing ECS Cluster configuration to host the fargate scheduled task. Defaults to creating its own cluster.                                                                   |                               |
+| schedule_expression           | string                          | When to execute this fargate task. Use 'cron()', 'rate()', or 'at()'. See [docs](https://docs.aws.amazon.com/scheduler/latest/UserGuide/schedule-types.html) for options.      | null                          |
```

Comment:
Isn't the "schedule_expression" a part of the "schedule" now?

Reply:
fixed